examples/hybrid/types.jl

```diff
@@ -130,16 +132,39 @@ function get_spaces(parsed_args, params, comms_ctx)
     dz_bottom = FT(parsed_args["dz_bottom"])
     dz_top = FT(parsed_args["dz_top"])
     topography = parsed_args["topography"]
+
+    if parsed_args["topography"] != "NoWarp"
+        include("topography_helper.jl")
+    end
+
     bubble = parsed_args["bubble"]
 
+    @assert topography in ("NoWarp", "DCMIP200", "Earth")
     if topography == "DCMIP200"
-        warp_function = CA.topography_dcmip200
+        warp_function = topography_dcmip200
     elseif topography == "NoWarp"
         warp_function = nothing
+    elseif topography == "Earth"
+        data_path = joinpath(topo_elev_dataset_path(), "ETOPO1_coarse.nc")
+        NCDataset(data_path) do data
+            zlevels = data["elevation"][:]
+            lon = data["longitude"][:]
+            lat = data["latitude"][:]
+            # Apply Smoothing
+            smooth_degree = 15
+            esmth = imfilter(zlevels, Kernel.gaussian(smooth_degree))
+            earth_spline = linear_interpolation(
+                (lon, lat),
+                zlevels,
+                extrapolation_bc = (Periodic(), Flat()),
+            )
+        end
+        @info "Generated interpolation stencil"
+        warp_function = generate_topography_warp(earth_spline)
```

Comment:
```suggestion
        warp_function = CA.generate_topography_warp(earth_spline)
```
(if that function is moved into `src/`

Reply:
Noted, thanks; leaving it in the `examples` directory for now; so I think I can do away with the `CA` parent module identifier